proxy/update_proxy.sh

```diff
@@ -49,9 +54,35 @@ function changelog_update {
     rm debian.changelog-bk
 }
 changelog_update $VERSION
-sed -e "s/@VERSION@/$VERSION/" cc-proxy.spec-template > cc-proxy.spec
-sed -e "s/@VERSION_DEB_TRANSFORM@/$VERSION_DEB_TRANSFORM/g;" -e "s/@HASH_TAG@/$short_hashtag/g;" cc-proxy.dsc-template > cc-proxy.dsc
-sed -e "s/@VERSION_DEB_TRANSFORM@/$VERSION_DEB_TRANSFORM/g;" -e "s/@HASH_TAG@/$short_hashtag/g;" debian.control-template > debian.control
+
+# Checkout OBS repo
+temp=$(basename $0)
+TMPDIR=$(mktemp -d -t ${temp}.XXXXXXXXXXX) || exit 1
+osc $APIURL co "$OBS_PROXY_REPO" -o $TMPDIR
+
+# Check for debian revision
+pushd $TMPDIR
+# The following line has to be removed/changed once "beta" is removed from version
+# in order to match the awk rule
+PACKAGED_VERSION=$(cat cc-proxy.dsc | awk -F [:-] '/^Version/ {gsub(/ /, "", $0); print $2"-"$3}')
+REVISION=$(cat cc-proxy.dsc | awk -F [:-] '/^Version/ {print $4}')
```

Comment:
Better to create a file called release like kernel does 

https://github.com/clearcontainers/packaging/blob/master/kernel/release

Reply:
Yes, that's better.